public/css/clipboard.css

```diff
@@ -0,0 +1,3 @@
+.fa-clipboard:hover {
+    color: gray;
+}
```

Comment:
Only other suggestion is add the following so the cursor changes when hovering:

```css
cursor: pointer;
```

Reply:
Done - thank you